CHANGELOG.next.asciidoc

```diff
@@ -15,6 +15,47 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Filebeat*
 
+- Convert netflow input to API v2 and disable event normalisation {pull}37901[37901]
```

Comment:
broken changelog merge should be fixed (along with the explicit broken merges below)

Reply:
Thanks! It looks like things were different back in the 7.x days. I had to change some imports to use the common package as well.